pysparkle/pysparkle/logger.py

```diff
@@ -4,6 +4,17 @@
 
 
 def setup_logger(name: str = "", log_level: int = logging.INFO):
+    """
+    Set up a colored logger with customizable log level and formatting.
+
+    Args:
+        name (str, optional): The name of the logger. Defaults to an empty string.
```

Comment:
Can you please use type hints instead of types in docstrings? See **function_with_pep484_type_annotations**  vs **function_with_types_in_docstring**  in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
done